src/grecon/grecon.c

```diff
@@ -103,7 +103,7 @@ void grecon(struct grecon_conf* param,  const long dims1[DIMS], complex float* o
 		sens1 = cov1;
 	}
 
-	fftmod(DIMS, dims1, FFT_FLAGS, sens1, sens1);
```

Comment:
It works for the case where the sense maps do not have extra dimensions, but the image does:
```bash
./phantom -s 8 -k k1
./bart repmat 5 2 k1 k2
./ecalib -1 k1 s1 # Dimensions 1 12 12 36
 # change sec = True in rsense. we should add a command line option here
./rsense k1 s1 o1
./rsense k2 s1 o2
./bart repmat 5 2 o1 o1.2
./bart relnorm o1.2 o2 # returns +0.000000e+00
```